custom_components/waste_collection_schedule/waste_collection_schedule/source/apps_ci_minneapolis_mn_us.py

```diff
@@ -0,0 +1,120 @@
+from datetime import datetime
+import logging
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.SSLError import get_legacy_session
+from waste_collection_schedule.exceptions import (
+    SourceArgumentException,
+    SourceArgumentRequired,
+)
+
+TITLE = "Minneapolis MN USA" # Title will show up in README.md and info.md
+DESCRIPTION = "Source script for apps_ci_minneapolis_mn_us"  # Describe your source
+URL = "https://www.minneapolismn.gov/resident-services/garbage-recycling-cleanup/collection-day-lookup/"  # Insert url to service homepage. URL will show up in README.md and info.md
```

Comment:
Please only use the main URL not the path to the lookup here.

Reply:
e.g. `https://www.minneapolismn.gov/`?

Reply:
yes as it shows up in the Readme.md and using the full url would take up more than one line a lot of the times.

Reply:
got it- updated